### PR TITLE
CVE-2004-0811

### PR DESCRIPTION
**Pull request by well-mannered-rascal**

### CODE REVIEW

File: cves/CVE-2004-0811.yml [hunk 1]
```diff
@@ -76,9 +81,9 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: false
```
Comment (JakeBash) [inline cves/CVE-2004-0811.yml]: I think for the "answer" field you are supposed to give a description as to what the state of unit testing was at the time of the fix.

Answer field from Meneely's example: "From looking at the fix code it's clear that code was tested and they added a new test for fixing this vulnerability."

File: cves/CVE-2004-0811.yml [hunk 2]
```diff
@@ -95,9 +100,9 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
+  answer: Discovered by a user running a simple test script to exercise a theory they had regarding request authentication.
+  date: 2002-11-21
+  automated: false
   google:
```
Comment (JakeBash) [inline cves/CVE-2004-0811.yml]: Maybe add false for the "google" and "contest" fields just for more clarity for a reader

File: cves/CVE-2004-0811.yml (hunk 1 above)
Comment (well-mannered-rascal) [reply cves/CVE-2004-0811.yml]: Thanks, that makes sense.

File: cves/CVE-2004-0811.yml (hunk 2 above)
Comment (well-mannered-rascal) [reply cves/CVE-2004-0811.yml]: Sure, sounds reasonable.

**Comment by JakeBash** [comment]
I give this 4 upvotes

**Pull request merged**